src/humancellatlas/data/metadata/api.py

```diff
@@ -290,20 +305,27 @@ def __init__(self, json: JSON) -> None:
 @dataclass(init=False)
 class CellSuspension(Biomaterial):
     estimated_cell_count: Optional[int]
-    selected_cell_type: Set[str]
+    selected_cell_types: Set[str]
 
     def __init__(self, json: JSON) -> None:
         super().__init__(json)
         content = json.get('content', json)
         self.estimated_cell_count = lookup(content, 'estimated_cell_count', 'total_estimated_cells', default=None)
-        self.selected_cell_type = {ontology_label(sct) for sct in content.get('selected_cell_type', [])}
+        self.selected_cell_types = {ontology_label(sct) for sct in
+                                    lookup(content, 'selected_cell_types', 'selected_cell_type', default=[])}
 
     @property
     def total_estimated_cells(self) -> int:
         warnings.warn(f"CellSuspension.total_estimated_cells is deprecated. "
                       f"Use CellSuspension.estimated_cell_count instead.", DeprecationWarning)
         return self.estimated_cell_count
 
+    @property
+    def selected_cell_type(self) -> int:
+        warnings.warn(f"CellSuspension.selected_cell_type is deprecated. "
+                      f"Use CellSuspension.selected_cell_types instead.", DeprecationWarning)
+        return self.selected_cell_types
```

Comment:
This makes no sense because it would return a set instead of a single string and clients would still crash. The point of a backwards compatibility wrapper is to provide a sensible degradation so client's don't fail. I think the best we can do here is 

```suggestion
        return one(self.selected_cell_types)
```

Reply:
The name of the property was changed from singular to plural, however the property was always an array. https://github.com/HumanCellAtlas/metadata-schema/pull/942/files